Cookbook/Technical-Documents/XcodeTips.md

```diff
@@ -0,0 +1,11 @@
+## Tips and tricks
+
+### Errors when pulling code/switching branches
+Various errors can occur when pulling code/switching branches. Typically all that is needed is running `pod install` and closing and reopening the project in XCode. The following directions are intended to be a more thorough reset to deal with bigger changes.
+
+- Clean Build: Product > Clean (shortcut: cmd + shift + k)
+- Close Xcode
+- Update bundle: run `bundle install`
+- Update pods: `pod repo update && pod deintegrate && pod install`
```

Comment:
We have git hooks that are checking if pods need to be updated after switching to another branch or pulling/merging (in `/githooks`). We can as well add it to check for bundler to be in sync

```suggestion
- Update pods: `pod deintegrate && pod install --repo-update`
```

Reply:
Don't we need to `bundle exec` ?

Reply:
Good call. I think it depends on one's setup. I don't need it on my machine, but adding bundle exec should be the safer option and work regardless. I will go ahead and add it.